src/rust/ensogl/lib/shape-utils/src/lib.rs

```diff
@@ -1,7 +1,4 @@
-//! ListView EnsoGL Component.
```

Comment:
The only thing I really dislike here is the name `shape-utils`. Basically we banned long time ago any module with a name `utils`, as it grows over the time to contain just everything. Utils is so brad name that it doesnt explain its purpose, so I dont even know what it should contain and why. If this module has utilities for dealing with shapes, maybe it should be just there where shapes are defined?

Reply:
I think it probably should be an extra crate, as it used non-core functionality. We might get circular dependencies if we add it to the `core` crate. Essentialy it is meant to provide functionality that build on shapes and other parts of our ecosystem. 
Maybe a better name would be `shape-extensions` ? I'm very open to suggestions.